client/src/components/Footer/style.css

```diff
@@ -0,0 +1,25 @@
+.footer {
+    display: flex;
+    flex-direction: row;
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100vw;
+    justify-content: space-around;
+    background: #F7F7F7;
+}
+
+.footer-element img {
+    width: 40px;
+    margin: auto;
+}
+
+.footer-element {
+    padding-top: 15px;
+    display: flex;
+    flex-direction: column;
+}
+
+.footer-element p {
+    margin: 5px auto 10px auto;
+}
```

Comment:
Use `BEM` styling coding because that what we agreed to use in our project

Reply:
I do it :)